packages/dev/core/src/Engines/Native/nativeInterfaces.ts

```diff
@@ -87,6 +87,31 @@ interface INativeEngineConstructor {
 
     readonly PROTOCOL_VERSION: number;
 
+    readonly CAPS_LIMITS_MAX_DRAW_CALLS: number;
+    readonly CAPS_LIMITS_MAX_BLITS: number;
+    readonly CAPS_LIMITS_MAX_TEXTURE_SIZE: number;
+    readonly CAPS_LIMITS_MAX_TEXTURE_LAYERS: number;
+    readonly CAPS_LIMITS_MAX_VIEWS: number;
+    readonly CAPS_LIMITS_MAX_FRAME_BUFFERS: number;
+    readonly CAPS_LIMITS_MAX_FB_ATTACHMENTS: number;
+    readonly CAPS_LIMITS_MAX_PROGRAMS: number;
+    readonly CAPS_LIMITS_MAX_SHADERS: number;
+    readonly CAPS_LIMITS_MAX_TEXTURES: number;
+    readonly CAPS_LIMITS_MAX_TEXTURE_SAMPLERS: number;
+    readonly CAPS_LIMITS_MAX_COMPUTE_BINDINGS: number;
+    readonly CAPS_LIMITS_MAX_VERTEX_LAYOUTS: number;
+    readonly CAPS_LIMITS_MAX_VERTEX_STREAMS: number;
+    readonly CAPS_LIMITS_MAX_INDEX_BUFFERS: number;
+    readonly CAPS_LIMITS_MAX_VERTEX_BUFFERS: number;
+    readonly CAPS_LIMITS_MAX_DYNAMIC_INDEX_BUFFERS: number;
+    readonly CAPS_LIMITS_MAX_DYNAMIC_VERTEX_BUFFERS: number;
+    readonly CAPS_LIMITS_MAX_UNIFORMS: number;
+    readonly CAPS_LIMITS_MAX_OCCLUSION_QUERIES: number;
+    readonly CAPS_LIMITS_MAX_ENCODERS: number;
+    readonly CAPS_LIMITS_MIN_RESOURCE_CB_SIZE: number;
+    readonly CAPS_LIMITS_TRANSIENT_VB_SIZE: number;
+    readonly CAPS_LIMITS_TRANSIENT_IB_SIZE: number;

```

Comment:
Are these all being used?

Reply:
No, we currently only use CAPS_LIMITS_MAX_TEXTURE_LAYERS, but I'm exposing all limits provided by bgfx in case the native engine needs any of them in the future. This way we don't have to change the PROTOCOL_VERSION again.

Reply:
I'm not sure about this. My guess is that you will have to update the protocol version anyways as this list may not translate correctly.

Reply:
I've updated the exposed caps to only have the ones we currently require or have a direct equivalent on the js caps.